mkinitcpio

```diff
@@ -364,6 +364,8 @@ process_preset() (
 
     . "$preset" || die "Failed to load preset: \`%s'" "$preset"
 
+    (( ! ${#PRESETS[@]} )) && warning "Preset file \`%s' does not contain any presets." "$preset"
```

Comment:
I think this is slightly better.
```suggestion
    (( ! ${#PRESETS[@]} )) && warning "Preset file \`%s' is empty or does not contain any presets." "$preset"
```

Reply:
Hi @grazzolini ,
agreed, sounds better.
Maybe, even better than that: should we refere to the `PRESETS` as variable, maybe?
Like `Preset file \`%s' is empty or does not define a value for PRESETS`. Even more specific and more clues what to look for.